bangazonapi/views/paymenttype.py

```diff
@@ -52,7 +52,8 @@ def retrieve(self, request, pk=None):
             Response -- JSON serialized payment_type instance
         """
         try:
-            payment_type = Payment.objects.get(pk=pk)
+            customer = Customer.objects.get(user=request.auth.user)
+            payment_type = Payment.objects.get(pk=pk, payment_type__customer=customer)
```

Comment:
I'm getting this error when I try to get a single payment type`Cannot resolve keyword 'payment_type' into field. Choices are: account_number, create_date, customer, customer_id,
deleted, expiration_date, id, merchant_name, order`

Reply:
Update the line 56 was for the single payment type was the filter wrong.